docs/sources/koji.rst

```diff
@@ -77,6 +78,21 @@ Push items produced in this case may include:
 - :class:`~pushsource.OperatorManifestPushItem` - if images have attached operator
   manifest archives
 
+Accessing virtual machine images
+...............................
+
+Use the ``vmi_build`` parameter to request virtual machine images from one or
+more builds. Builds can be specified by NVR. Builds should be produced by
+`Image Builder`_ or should use compatible metadata.
+
+``koji:https://koji.fedoraproject.org/kojihub?vmi_build=fedora-azure-37-1.7.x86_64``
```

Comment:
I think the arch should still not be there, right?  In practice does arch usually form part of the "release"?  

It doesn't seem like it; for example, I found a build named `rhel-azure-7.9-20230110.sp.22`.

```suggestion
``koji:https://koji.fedoraproject.org/kojihub?vmi_build=fedora-azure-37-1.7``
```

Reply:
I've added your suggestion, PTAL again